src/components/app/SingleStrategy/index.tsx

```diff
@@ -16,24 +16,23 @@ import StrategyDetail from './StrategyDetail';
 import StrategyHealthCheck from './StrategyHealthCheck';
 
 import { ErrorAlert } from '../../common/Alerts';
-import { Strategy, Vault, Network, DEFAULT_NETWORK } from '../../../types';
+import { Network, DEFAULT_NETWORK } from '../../../types';
 
 import BreadCrumbs from './BreadCrumbs';
 import EtherScanLink from '../../common/EtherScanLink';
 import ReactHelmet from '../../common/ReactHelmet';
 import ProgressSpinnerBar from '../../common/ProgressSpinnerBar/ProgressSpinnerBar';
 
-import { getService as getVaultService } from '../../../services/VaultService';
-import {
```

Comment:
I think it's probably cleaner to keeping doing the imports all from the same base `utils` location